aten/src/ATen/native/cuda/RreluWithNoise.cu

```diff
@@ -132,6 +132,10 @@ Tensor& rrelu_with_noise_out_cuda(const Tensor& self,
     bool training,
     c10::optional<Generator> generator,
     Tensor& output) {
+  if (noise.numel() == 0) {
+    return output;
```

Comment:
is output guaranteed to have the right shape/type here?  Doesn't seem like it because of the checks below.

Reply:
@gchanan 
that output made by `empty_like`
So if input tensor is empty, then return empty tensor same as input

but I'm not considered wrong(sounds weird, I mean not normal and common cases) tensor

Reply:
This is public api, which means that `output` tensor can have wrong shape here, so you need to check it before returning

Reply:
@zsef123 as described [here](https://github.com/pytorch/pytorch/blob/8aa3620d73d9afcdbcf3d691cdd45ae05ec5e7d5/aten/src/ATen/native/Resize.h#L12-L24), `at::native::resize_output(output, self.sizes());` should be called, which will resize output and emit a warning if it was the incorrect (non-empty) size.